ivy_tests/test_ivy/test_functional/test_core/test_statistical.py

```diff
@@ -124,6 +125,8 @@ def test_max(*, dtype_and_x, keep_dims, test_flags, backend_fw, fn_name, on_devi
 )
 def test_mean(*, dtype_and_x, keep_dims, test_flags, backend_fw, fn_name, on_device):
     input_dtype, x, axis = dtype_and_x
+    tolerance_dict = defaultdict(lambda: 1e-2)
```

Comment:
Lets remove this since it adds unnecesary code, we can just pass `{"bfloat16": 1e-1}`
directly in the `tolerance_dict` argument below